server/src/com/thoughtworks/go/server/service/PipelineConfigService.java

```diff
@@ -132,6 +136,17 @@ public void updatePipelineConfig(final Username currentUser, final PipelineConfi
         update(currentUser, pipelineConfig, result, updatePipelineConfigCommand);
     }
 
+    public List<Pipeline> pipelinesFor(Username username) {
```

Comment:
Check 
https://github.com/gocd/gocd/blob/e43774e9f991c2a0e2c82a3e41feeaa01a8efcbe/server/webapp/WEB-INF/rails.new/app/controllers/api_v1/dashboard_controller.rb

for an example.